src/polar/RadialBar.tsx

```diff
@@ -398,14 +397,7 @@ export class RadialBar extends PureComponent<Props, State> {
 
         <Layer className="recharts-radial-bar-sectors">{this.renderSectors()}</Layer>
 
-        {(!isAnimationActive || isAnimationFinished) &&
-          LabelList.renderCallByParent(
-            {
-              ...this.props,
-              clockWise: this.getDeltaAngle() < 0,
```

Comment:
why was `clockWise` removed? Unless there is a reason if should probably remain

Reply:
> On RadialBar I removed clockWise prop from LabelList.renderCallByParent invocation since the function uses only label and children props

from the description

Reply:
Ah sorry I missed that